torchgeo/models/rcf.py

```diff
@@ -104,3 +107,111 @@ def forward(self, x: Tensor) -> Tensor:
             assert len(x1a.shape) == 2
             output = torch.cat((x1a, x1b), dim=1)
             return output
+
+
+class MOSAIKS(RCF):
+    """This model extracts MOSAIKS features from its input.
+
+    MOSAIKS features are described in Multi-task Observation using Satellite Imagery &
+    Kitchen Sinks https://www.nature.com/articles/s41467-021-24638-z. Briefly, this
```

Comment:
Should we use a link here? Would like the format of RCF and MOSAIKS to match. What if we instead use something like:
```rst
If you use this model in your research, please cite the following paper:

* https://www.nature.com/articles/s41467-021-24638-z
```